docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,279 @@
+
+## 4. Configuring microk8s
+
+1. First, we must enable the following plugins on microk8s:
+   1. `sudo microk8s.enable dns && sudo microk8s.status --wait-ready`
+   2. `sudo microk8s.enable storage && sudo microk8s.status --wait-ready`
+   3. `sudo microk8s.enable ingress &&  sudo microk8s.status --wait-ready`
+2. Now that these plugins have been enabled, we should restart microk8s to ensure a proper bootstrap of the cluster: `microk8s.stop && microk8s.start && microk8s.status --wait-ready`. Further information on this subject can be [found here](https://github.com/ubuntu/microk8s/issues/493#issuecomment-498167435).
+3. The plugins are now enabled and the cluster bootstrapped. However, we must still wait for some microk8s internals (dns, http, and ingress) plugins to be ready. Failing to do so can result in pods entering a `CrashLoopBackoff` state:
+
+   1. ```
+      microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=kube-dns --timeout=300s &&
+      microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=hostpath-provisioner --timeout=300s &&
+      microk8s.kubectl wait -n default --for=condition=Ready pod -l app=default-http-backend --timeout=300s &&
+      microk8s.kubectl wait -n default --for=condition=Ready pod -l name=nginx-ingress-microk8s --timeout=300s
+      ```
+
+After runing these commands, we have ensured that dns, http, and nginx ingress are enabled and working properly inside the cluster.
+
+## 5. Installing Codacy
+
+### 1. Helm and Tiller
+
+First we must install `helm` onto the cluster. With `helm`, we can easily install charts and manage the lifecycle of the installed artifacts. One example is the ability to rollback a failed install.
+1. Assuming you would want to install `helm` version 2.16.3, export the following environment variable: `export HELM_VERSION=2.16.3`
+   1. Please note that we currently do *not* support helm v3.
+2. Run this script that will retrieve the desired `helm` version and install it onto the machine's `/usr/local/bin`: `HELM_PKG=helm-${HELM_VERSION}-linux-amd64.tar.gz
+wget https://get.helm.sh/$HELM_PKG
+tar xvzf $HELM_PKG
+su mv linux-amd64/tiller linux-amd64/helm /usr/local/bin
+rm -rvf $HELM_PKG linux-amd64/`.
+4. We must now deploy `tiller` onto the cluster.
+   1. First we must set a `serviceaccount` and a `clusterrolebinding` for `tiller` on the microk8s cluster: `microk8s.kubectl create serviceaccount --namespace kube-system tiller
+microk8s.kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller`
+   2. Then we can install `tiller`, which will be installed with the same `HELM_VERSION` that you have specified earlier: `helm init --service-account tiller`
+   3. We wait for tiller to be ready: `microk8s.kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller --timeout=300s`
+
+### 2. Codacy chart
```

Comment:
We should make use of the other docs as much as possible and put only the diffs here: https://github.com/codacy/chart/blob/master/docs/install.md

Reply:
this sections needs to be as little as possible